src/operations/functions/createFunction.ts

```diff
@@ -64,6 +65,16 @@ export function createFunction(mOptions: MigrationOptions): CreateFunction {
       options.push(`PARALLEL ${parallel}`);
     }
 
+    if (set) {
+      for (const { name, value } of set) {
+        if (value === 'FROM CURRENT') {
+          options.push(`SET ${name} FROM CURRENT`);
+        } else {
+          options.push(`SET ${name} TO ${value}`);
```

Comment:
**question (if-minor):** do we need `mOptions.literal` or `escapeValue` here?

Reply:
Please try out if we can benefit still from `mOptions.literal(configurationParameter)` as this could escape the user's input and secure it a bit.
Maybe if you have a local PostgreSQL, test out there first and play a bit around with it. `mOptions.literal` quotates the input from the user.

Reply:
Looking into it.

Reply:
So I did a bit of research and this is what I found out:

1. The `configurationParameter` must be a ["settable run-time parameter"](https://www.postgresql.org/docs/current/sql-set.html). We can safely quote them. 
2. The value however can be ["of one of five types: boolean, string, integer, floating point, or enumerated (enum). The type determines the syntax for setting the parameter"](https://www.postgresql.org/docs/current/config-setting.html#CONFIG-SETTING-NAMES-VALUES). I think we can use single quotes here or omit quotes completely.